src/e2e/puppeteer/__tests__/caret-multiline.ts

```diff
@@ -263,4 +263,27 @@ describe('all platforms', () => {
     const offset = await getSelection().focusOffset
     expect(offset).toBe(0)
   })
+
+  // test case 11
+  it('on cursorUp, the caret should move from the start of the 2nd line in a multi-line cursor to the start of the 1st line in the same cursor.', async () => {
+    const multiLineCursor =
+      "Beautiful antique furnishings fill this quiet, comfortable flat across from the Acropolis museum. AC works great. It is in an heavily touristic area, but the convenience can't be beat. Highly recommended."
+    const importText = `
+  - a
+  - ${multiLineCursor}`
+
+    await paste(importText)
+
+    const editableNodeHandle = await waitForEditable(multiLineCursor)
+    await click(editableNodeHandle, { y: 1 })
```

Comment:
`y` is specified in pixels, so this would click on the editable at (1,1), which I think would place the selection at the start of the 1st line, no?

Yet the caret is somehow still on the multiline thought after ArrowUp. What am I missing?

Note: You could ArrowDown first to get the caret onto the second line.

Reply:
Oh, I guess I misunderstood how it worked a little bit. From `helpers/click.ts`:

```
  const coordinate = !offset
    ? {
        x: boundingBox.x + (edge === 'left' ? 1 : edge === 'right' ? boundingBox.width - 1 : boundingBox.width / 2),
        y: boundingBox.y + boundingBox.height / 2,
      }
    : await offsetCoordinates()
```

So it's clicking on the left of the bounding box, halfway down plus 1 pixel. Which in my case is the correct position. It sounds like ArrowDown is the way to go, thanks for pointing that out.

Reply:
You're right!!! My bad. So it would click on (1, height/2 + 1). This would indeed click on the second line IF the thought wraps in the correct place. 

It would be great to make this test independent of the width of the browser window as we did in the other tests.

Reply:
I updated it so that it uses ArrowDown. There are some test errors that appear unrelated to anything I changed.